lib/full-site-editing/template-loader.php

```diff
@@ -79,8 +79,16 @@ function gutenberg_override_query_template( $template, $type, array $templates =
 	}
 
 	if ( $current_template ) {
-		$_wp_current_template_content = empty( $current_template->content ) ? __( 'Empty template.', 'gutenberg' ) : $current_template->content;
-
+		if ( empty( $current_template->content ) && isset( $current_template->title ) && is_user_logged_in() ) {
```

Comment:
I like the addition of showing the title but I'm not sure if there is a flow that could lead to no `$current_template->title`. If there isn't we should skip this check. If there is, we should still show a message, no?